src/styles/uiConfigurations.css

```diff
@@ -1,6 +1,6 @@
 .toolbar {
   /* display: none; */
-  height: max-content;
+  height: 30px;
```

Comment:
Toolbar height is set to max-content to accommodate expansion when we click on the font-size/colour buttons. Fixing the height will break that.